docs/nitpick-exceptions

```diff
@@ -38,31 +27,19 @@ py:class yaml.resolver.Resolver
 py:class yaml.emitter.Emitter
 
 # astropy.units
-py:obj astropy.units.function.logarithmic.m_bol
+py:obj astropy.units.function.logarithmic.m_bol  # Required for Windows
```

Comment:
```suggestion
py:obj astropy.units.function.logarithmic.m_bol  # Required for Windows and OSX
```

Reply:
Has this been tested on macOS?

Reply:
Not by me, but it is a known long-standing issue on both the OSes. If you have a Mac, please do test it. Thanks! I do not have one.

Reply:
If I am supposed to write down that this exception is required also by macOS then I really want this to be tested first, and I can't test this myself. I am very hesitant to guess what the behavior on macOS is without actually testing it. My impression is that whatever ideas you might have about how macOS works, the reality is you should think different.

Reply:
I believe the OSX case is documented here:

* https://github.com/astropy/astropy/issues/9040

Reply:
That's good enough for me.